test/k8sT/Policies.go

```diff
@@ -892,6 +892,20 @@ EOF`, k, v)
 
 		})
 
+		waitServiceBackendsPlumbed := func() {
+			for _, ns := range []string{helpers.DefaultNamespace, secondNS} {
+				By("Waiting for backends to be plumbed for %s in %s namespace", app1Service, ns)
+				err = kubectl.WaitForServiceEndpoints(
+					ns, "", app1Service,
+					helpers.HelperTimeout)
+				ExpectWithOffset(1, err).Should(BeNil(), "Service %q in namespace %q is not ready after timeout", app1Service, ns)
+			}
+		}
+
+		BeforeEach(func() {
+			waitServiceBackendsPlumbed()
```

Comment:
Why not `kub.waitForNPods(checkReady, namespace, filter, minRequired, timeout)`?

Reply:
The service backends will only be plumbed if the pods themselves are ready. Given that we are testing connectivity via these services, it makes more sense to me to check for the services being ready, since that implies the pods are ready as well.

Reply:
Looking at the code a bit deeper, I'd like to add a check that both the backends are plumbed for the service. I'm not sure the current test code can check for that. I can add that as a follow-up commit.